scp.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"os"
+	"time"
+
+	scp "github.com/bramvdbogaerde/go-scp"
+	"github.com/bramvdbogaerde/go-scp/auth"
+	"golang.org/x/crypto/ssh"
+)
+
+func scpToWorker(workerURL string, source string, dest string, tarName string) error {
+	// Use SSH key authentication from the auth package
+	// we ignore the host key in this example, please change this if you use this library
+	clientConfig, err := auth.PrivateKey("ubuntu", "/home/ubuntu/.ssh/senior-design.pem", ssh.InsecureIgnoreHostKey())
+	if err != nil {
+		Error.Println("Error creating ssh config", err)
+		return err
+	}
+
+	// Create a new SCP client
+	workerURL += ":22"
+	// Timeout after 5 minutes
+	client := scp.NewClientWithTimeout(workerURL, &clientConfig, time.Duration(300000000000))
```

Comment:
there has to be a clearer way to write `5 minutes`

Reply:
I could define a minute as 60000000000 nanoseconds somewhere else and then write 5*minute if that's cleaner

Reply:
Either use `time.ParseDuration` or `5 * time.Minute`

Reply:
will do